config/default.yml

```diff
@@ -212,8 +212,9 @@ RSpec/ContextWording:
     - when
     - with
     - without
+  AllowedPatterns: []
```

Comment:
Should we have default allowed patterns, matching the prefixes, and deprecate the prefixes option?

Reply:
I am certainly beginning to think that is a good idea.
@bquorning @pirj What do you think?